org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -582,6 +605,9 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
                 case ZigBeeBindingConstants.CONFIGURATION_LEAVE:
                     coordinatorHandler.leave(nodeIeeeAddress);
                     break;
+                case ZigBeeBindingConstants.CONFIGURATION_INITIALIZE_DEVICE:
+                    initializeDevice = true;
```

Comment:
I'm wondering if I need to check the value of the config param instead of just setting the variable to `true`

Reply:
I don't see why? 
If it's previously false, you set it to true so it initialises as requested. If it was previously true, then you will still initialise as requested.